h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -54,7 +54,8 @@ function TextField({
   type,
   value,
   onChangeValue,
-  limit,
+  minLength = -1,
```

Comment:
Using `-1` rather than `null` or `undefined` as the default value makes the code nicer because you can just do things like `if (value.length < minLength)` or `>= minLength` without always having to test for `=== null` or `!== null` first.

Reply:
Done